### PR TITLE
Chore/change resourcemap index

### PR DESCRIPTION
**Pull request by ankitnayan**
change the type of index for resource attributes from `bloom_filter` to `tokenbf_v1` for keys and `ngrambf_v1` for values. This is tested and implemented in logs already.

### CODE REVIEW

**Comment by ankitnayan** [comment]
Merge https://github.com/SigNoz/signoz-otel-collector/pull/367 before merging this

File: migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql
Comment (srikanthccv) [inline migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql:3]: Can you drop the `DROP INDEX` from this migration and use different name for tokenbf indexes? `DROP INDEX` is implemented as mutation and will likely fail the migration and lead to dirty migration issues.

Comment (srikanthccv) [reply migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql:3]: We can DROP INDEXES later when we have the schema migrator ready.

Comment (ankitnayan) [reply migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql:3]: the names are different. I changed to keep the name synced to the column name. New is `idx_resourceTagsMapKeys` an old was `idx_resourceTagMapKeys`. Notice the `Tags` vs `Tag`.

Do you want to remove the `DROP INDEX ...` from up file only or also removing `Add INDEX ...` in the down file?

Comment (srikanthccv) [reply migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql:3]: Ah, I didn't notice that. Let's just remove the `DROP INDEX` from up migration file.

Comment (ankitnayan) [reply migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql:3]: 2 indexes on a column should be fine?

Comment (srikanthccv) [reply migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql]: Yes, it should be fine. We will incur some costs for index storage but from CH will pick the best one out of all created indexes.

**Review by srikanthccv — APPROVED** [review]
Will wait for other PR to get merged

**Comment by ankitnayan** [comment]
> Merge #367 before merging this

this is not a blocker. The sequencing of migration files look fine

**Pull request merged**
